sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
It was not provided, but I am afraid of the version of this leaking into the users of the common library. What do you think should I let it like it was (without the provided) ?